### PR TITLE
[AIRFLOW-3612] Remove incubation/incubator mention

### PR DESCRIPTION
**Pull request by feng-tao**
Make sure you have checked _all_ steps below.

### Jira

- [ ] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-3612
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:
Airflow moved to TLP. Remove all the "incubation/incubator" mentioning.

### Tests

- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [ ] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.
  - All the public functions and the classes in the PR contain docstrings that explain what it does

### Code Quality

- [ ] Passes `flake8`

### CODE REVIEW

**Comment by feng-tao** [comment]
PTAL @ashb @kaxil @Fokko @bolkedebruin @mistercrunch @r39132 @jghoman

Please take a look in case I miss anything.

**Comment by Rotemlofer** [comment]
Does the git-hub repository going to be changed as well apache/incubator-airflow   ?

**Comment by feng-tao** [comment]
@Rotemlofer, I believe it will change once the TLP announcement is official.

**Comment by mistercrunch** [comment]
Side note, we'll have to open a Jira with Apache infra for them to edit this:
screen shot 2019-01-02 at 1 12 34 pm

**Comment by feng-tao** [comment]
PTAL @ashb @kaxil @Fokko @bolkedebruin @mistercrunch @r39132 @jghoman

I believe we need to commit this change by 1/8?

**Comment by feng-tao** [comment]
Plus @jgao54  as well

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -261,14 +261,14 @@ Feel free to customize based on the extras available in [setup.py](./setup.py)
...
+1. As Airflow grows as a project, we try to enforce a more consistent style and try to follow the Python community guidelines. We track this using [landscape.io](https://landscape.io/github/apache/airflow/), which you can setup on your fork as well to check before you submit your PR. We currently enforce most [PEP8](https://www.python.org/dev/peps/pep-0008/) and a few other linting rules. It is usually a good idea to lint locally as well using [flake8](https://flake8.readthedocs.org/en/latest/) using `flake8 airflow tests`. `git diff upstream/master -u -- "*.py" | flake8 --diff` will return any changed files in your branch that require linting.
```
Comment (kaxil) [inline CONTRIBUTING.md]: I don't think we use landscape anymore

File: setup.py [hunk 2]
```diff
@@ -410,10 +410,10 @@ def do_setup():
             'Topic :: System :: Monitoring',
         ],
         author='Apache Software Foundation',
-        author_email='dev@airflow.incubator.apache.org',
-        url='http://airflow.incubator.apache.org/',
+        author_email='dev@airflow.apache.org',
+        url='http://airflow.apache.org/',
         download_url=(
-            'https://dist.apache.org/repos/dist/release/incubator/airflow/' + version),
+            'https://dist.apache.org/repos/dist/release//airflow/' + version),
```
Comment (kaxil) [inline setup.py]:
```suggestion
            'https://dist.apache.org/repos/dist/release/airflow/' + version),
```

**Review by kaxil — CHANGES_REQUESTED** [review]
Minor changes suggested

**Review by kaxil — APPROVED** [review]

**Comment by kaxil** [comment]
I made 2 changes that were needed.

**Comment by kaxil** [comment]
I am merging this one. There are still places that needs changing which I am doing it in a separate PR.

**Pull request merged**